### ISSUE TITLE
Update cluster configuration docs so it is more obvious to use different gossip-port values for each instance on the same host

### ISSUE THREAD

**Issue by yuce** [pilosa/pilosa#871]

**Comment by jaffee** [comment]
should we go ahead and close this @yuce ? or do you think we should add an example of single node cluster configuration?

**Comment by yuce** [comment]
@jaffee I've pushed a PR before which made things even more confusing so I've closed it. Adding the single node cluster example is a good idea, I'll create a new PR with it.

**Comment by yuce** [comment]
This is blocked by https://github.com/pilosa/pilosa/pull/889 which changes `-gossip-port` to `--gossip.port`.

**Comment by yuce** [comment]
Fixed by https://github.com/pilosa/pilosa/pull/901, closing.
